src/tarantool.client/Model/TarantoolTuple.cs

```diff
@@ -30,26 +31,35 @@ public override bool Equals(object obj)
 
         public override int GetHashCode()
         {
-            var hashCode = EqualityComparer<T1>.Default.GetHashCode(Item1);
-            return hashCode;
+            unchecked
```

Comment:
@roman-kozachenko in this class - no we do not need. But we need it in other classes. So, as this code is autogenerated, I will not edit it.